src/main/java/frc/team2412/robot/commands/autonomous/TwoBallAutoCommandLeft.java

```diff
@@ -16,33 +16,30 @@
 import frc.team2412.robot.commands.shooter.ShooterTargetCommand;
 import frc.team2412.robot.subsystem.DrivebaseSubsystem;
 import frc.team2412.robot.subsystem.IndexSubsystem;
-import frc.team2412.robot.subsystem.IntakeSubsystem;
 import frc.team2412.robot.subsystem.ShooterSubsystem;
 import frc.team2412.robot.subsystem.ShooterVisionSubsystem;
 
 public class TwoBallAutoCommandLeft extends SequentialCommandGroup {
-    public TwoBallAutoCommandLeft(IndexSubsystem indexSubsystem, ShooterSubsystem shooterSubsystem,
-            ShooterVisionSubsystem shooterVisionSubsystem, DrivebaseSubsystem drivebaseSubsystem,
-            IntakeSubsystem intakeSubsystem) {
-        // Robot should be pressed up on the left side of the lower exit closest to the drivers, facing
+    /**
+     * Subsystems: {@link DrivebaseSubsystem}, {@link IndexSubsystem}, {@link ShooterSubsystem},
+     * {@link ShooterVisionSubsystem}
+     */
+    public TwoBallAutoCommandLeft() {
+        // Robot should be pressed up on the left side of the lower exit closest to the drivers,
+        // facing
         // directly away from the hub with the turret facing towards it
```

Comment:
If you want to join wrapped comments, you should join these two lines.

Reply:
Strange, I set formatter to automatically join lines